view/frontend/web/js/lib/algolia-search.min.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Skipped minification because the original files appears to be already minified.
+ * Original file: /npm/algoliasearch@5.1.1/dist/algoliasearch.umd.js
+ *
+ * Do NOT use SRI with dynamically generated files! More information: https://www.jsdelivr.com/using-sri-with-dynamic-files
+ */
+/*! algoliasearch.umd.js | 5.1.1 | © Algolia, inc. | https://github.com/algolia/algoliasearch-client-javascript */
```

Comment:
Version 5 of `algoliasearch` lib is brand new and though it's hard to tell the exact version of the bundle (because the original repo was never updated properly) we are most definitely on some flavor of [4.x](https://github.com/algolia/algoliasearch-extensions-bundle/blob/ISv4/package.json#L17). 

So my feeling here is that while we should update to v5 of the library as soon as possible, we should _not_ do so until we've successfully migrated off of `algoliaBundle`. I think it would be best at this stage of the journey to only update to the last stable version 4 of `algoliasearch` which appears to be [4.24.0](https://www.jsdelivr.com/package/npm/algoliasearch?version=4.24.0&tab=files). 

The reason for my thinking is that by doing this we minimize the variables of the migration and don't end up chasing our tails on something that could be related to the v5 upgrade and not actually the bundle migration. What do you think?

Reply:
I totally agree with this. Removing algoliaBundle is a big task and we want this to be smooth. We don't want to introduce additional challenges in this task. One step at a time make sense to me. Will fix it soon.